quirks/change-password-URLs.json

```diff
@@ -20,6 +20,7 @@
     "allianz.com.br": "https://www.allianz.com.br/alteracao-de-password-ecliente",
     "allrecipes.com": "https://www.allrecipes.com/account/profile#/change-password",
     "alternate.de": "https://www.alternate.de/html/myAccount/account/basicData.html",
+    "amazon.com": "https://www.amazon.com/ax/account/manage",
```

Comment:
Thank you @danielleflinn :)

If you'd like, this would be a good opportunity to add the equivalent urls for the rest of the global amazon domains. You can find a list in `websites-with-shared-credential-backends.json`.

Reply:
Great idea; added!